executor_manager/routers/routers.py

```
@@ -202,3 +215,115 @@ async def cancel_task(request: CancelTaskRequest, http_request: Request):
+                restart_limiter.clear_restart_count(request.subtask_id)
+            except Exception as e:
+                logger.warning(f"Failed to clear restart count for subtask {request.subtask_id}: {e}")
+
+            return ExecutorStartupResponse(
+                allowed=False,
+                restart_count=check_result.restart_count,
+                max_restart=check_result.max_restart,
+                message=error_message
+            )
+
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)